docs/content/code-outputs.md

```diff
@@ -246,353 +246,3 @@ sphinx:
       - "text/latex"
       - "text/plain"
 ```
-
-(content:code-outputs:glue)=
```

Comment:
This is just me splitting this off into it's own page so that it is more discoverable